Backpack/Classes/Font/Generated/BPKFont.h

```diff
@@ -19,16 +19,16 @@
 
 typedef NS_ENUM(NSUInteger, BPKFontStyle) {
 
-    BPKFontStyleTextBase,
-    BPKFontStyleTextBaseEmphasized,
-    BPKFontStyleTextLg,
-    BPKFontStyleTextLgEmphasized,
-    BPKFontStyleTextSm,
-    BPKFontStyleTextSmEmphasized,
-    BPKFontStyleTextXl,
-    BPKFontStyleTextXlEmphasized,
-    BPKFontStyleTextXs,
-    BPKFontStyleTextXsEmphasized,
+    BPKFontStyleTextBase = 0,
+    BPKFontStyleTextBaseEmphasized = 1,
+    BPKFontStyleTextLg = 2,
+    BPKFontStyleTextLgEmphasized = 3,
+    BPKFontStyleTextSm = 4,
+    BPKFontStyleTextSmEmphasized = 5,
+    BPKFontStyleTextXl = 6,
+    BPKFontStyleTextXlEmphasized = 7,
+    BPKFontStyleTextXs = 8,
+    BPKFontStyleTextXsEmphasized = 9,
```

Comment:
This is to have stability when used from IB. Because when you make an enum `IBInspectable` it can only be used as a raw number from IB so having them explicitly declared helps consumers.